task_hub/models.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Position(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Worker(AbstractUser):
+    position = models.ForeignKey(
+        Position,
+        on_delete=models.CASCADE,
+        related_name="workers"
+    )
+
+    class Meta:
+        verbose_name = "worker"
+        verbose_name_plural = "workers"
+        ordering = ["position", ]
+
+    def __str__(self):
+        return f"{self.position} ({self.first_name} {self.last_name})"
+
+
+class TaskType(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+    LOW = "4"
```

Comment:
for choice field use next structure 
<img width="934" alt="Screenshot 2023-10-20 at 17 18 42" src="https://github.com/o-davydova/IT-task-manager/assets/117740672/d3cc20e6-842e-42b1-a766-d8c39029d15e">